### DISCUSSION TITLE
[Request] Plausible LXC Container

### DISCUSSION THREAD

**Discussion by nacho-domatix** [tteck/Proxmox#3571] [Ideas]
I was looking for a way to install Plausible and it would be amazing to have an script to do it under Proxmox.

https://github.com/plausible/community-edition/

[Plausible](https://plausible.io) is an open source web analytics tool, similar to Google Analytics but free, self-hosted ready and AGPL-licensed.

Thank you so much for your work!

**Comment by Narkowii** [comment]
There is already a script for Umami [https://github.com/umami-software/umami](url)

**Comment by nacho-domatix** [comment]
Thanks for the tip. As other apps that can cover the same functionality, I'd rather go for Plausible insted of Umami. Hope it can be done with this scripts. :)
